get_winner.py

```diff
@@ -0,0 +1,60 @@
+"""Takes in a dictionary of votes, and returns the meme with the highest wins to contests."""
+
+
+from vote.vote import MemeVote
+
+
+def group_by(iterable, key):
+    """Place each item in an iterable into a bucket based on calling the key
+    function on the item."""
+    group_to_items = {}
+    for item in iterable:
+        group = key(item)
+        if group not in group_to_items:
+            group_to_items[group] = []
+        group_to_items[group].append(item)
+    return group_to_items
+
+
+def get_winner_meme(vote):
+    """Simple key for groupby function in get_meme_to_wins function."""
+    return vote['winner']
+
+
+def get_memes_to_wins(memes_to_votes):
+    memes_to_wins = {
+        meme: [len(votes)]
+        for meme, votes
+        in memes_to_votes
+        }
+    return memes_to_wins
```

Comment:
Why are the keys in this dictionary a list of a single int as opposed to just an int?